src/components/SignUp/SignUp.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import backgroundImageStyling from '../../styles/backgroundImageStyling'
+import {Button, Form} from "react-bootstrap";
+import './SignUp.css';
+import Image from "../../utils/Image";
+import {Link} from "react-router-dom";
+
+
+//require any images
+const logo = require('../../images/swarmLogoIcon.png');
+const background = backgroundImageStyling();
+
+class SignUp extends React.Component {
+    constructor(props: any) {
+        super(props);
+    }
```

Comment:
any, doesn't even look like it needs a constructor since there isn't a props interface